tests/std/tests/GH_002992_unwrappable_iter_sent_pairs/test.compile.pass.cpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <filesystem>
+#include <forward_list>
+#include <list>
+#include <map>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+#include <xutility>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+
+template <class Rng>
+void test_algorithms(Rng& rng) {
+    auto dist  = ranges::distance(rng);
+    auto other = vector<ranges::range_value_t<Rng>>(static_cast<size_t>(dist));

```

Comment:
I _hate_ direct initialization of locals with parentheses because it looks like a function declaration. I will change it but I'm not super happy about it...

Reply:
If only it weren't a `vector` (with overloaded `initializer_list` constructors), then we could use direct-list-init :crying_cat_face: